drivers/png/SCsub

```diff
@@ -57,6 +47,13 @@ if env["builtin_libpng"]:
         neon_sources.append(env_neon.Object(thirdparty_dir + "/arm/filter_neon.S"))
```

Comment:
Assembly files probably won't be compatible with MSVC build for ARM Windows, so should have `not env.msvc` checked.